lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -348,6 +365,42 @@ describe('LMSFilePicker', () => {
     assert.isNotOk(finalReloadButton.prop('disabled'));
   });
 
+  it('looks for inner children when reloading on a system supporting full tree files loading', async () => {
```

Comment:
I will do the second suggestion for now, if that's ok, just to avoid introducing more changes that may need further reviews.

I will do a smaller follow-up PR to refine that though. I also need to check what `context(...)` does, I didn't know that was a thing 😅